src/views/ProfileView.vue

```diff
@@ -24,7 +24,7 @@
         </router-link>
       </div>
       <div class="announcements-list__container">
-        <announcements-list />
+        <announcements-list sitewide count="10"/>
```

Comment:
what is "10"?

Reply:
should be a constant in utils, whatever it is.

Reply:
10 is the number of announcements to display. I'll make it a constant